netlify-plugin-cloudinary/src/lib/cloudinary.ts

```diff
@@ -5,7 +5,7 @@ import { JSDOM } from 'jsdom'
 import { v2 as cloudinary } from 'cloudinary'
```

Comment:
@matiasfha  can you check on line 330 of this file where the srcset is updated, can you verify an example with srcset will throw and catch the errors appropriately?

i'm not sure if we have a test for this and additionally may need to add it to the example being tested, but there's no try/catch around it like the other instnace of getCloudinaryUrl

https://github.com/cloudinary-community/netlify-plugin-cloudinary/blob/bca5432772e5cfe38874d2e029187d9f026bfe46/netlify-plugin-cloudinary/src/lib/cloudinary.ts#L330